integration-tests/mcms/common.go

```diff
@@ -444,4 +449,172 @@ func (s *MCMSTestSuite) ExecuteProposalE2e(timelockProposal *mcms.TimelockPropos
+	s.T().Logf("✅ Transferred ownership of CCIP Router to MCMS in tx: %s", tx.Digest)
+
+	// 1c. Transfer ownership of the CCIP OnRamp to MCMS
+	ccipOnRampContract, err := module_onramp.NewOnramp(s.ccipOnrampPackageId, s.client)
+	require.NoError(s.T(), err, "creating ccip onramp contract")
+
+	tx, err = ccipOnRampContract.TransferOwnership(
+		s.T().Context(),
+		&bind.CallOpts{
+			Signer:           s.signer,
+			WaitForExecution: true,
+		},
+		bind.Object{Id: s.ccipObjects.CCIPObjectRefObjectId},
+		bind.Object{Id: s.ccipOnrampObjects.StateObjectId},
+		bind.Object{Id: s.ccipOnrampObjects.OwnerCapObjectId},
+		s.mcmsPackageID,
+	)
+	require.NoError(s.T(), err, "transferring ownership of CCIP OnRamp to MCMS")
+	require.NotEmpty(s.T(), tx, "Transaction should not be empty")
+
+	s.T().Logf("✅ Transferred ownership of CCIP OnRamp to MCMS in tx: %s", tx.Digest)
+
+	// 1d. Transfer ownership of the CCIP OffRamp to MCMS
+	ccipOffRampContract, err := module_offramp.NewOfframp(s.ccipOfframpPackageId, s.client)
+	require.NoError(s.T(), err, "creating ccip offramp contract")
+
+	tx, err = ccipOffRampContract.TransferOwnership(
+		s.T().Context(),
+		&bind.CallOpts{
+			Signer:           s.signer,
+			WaitForExecution: true,
+		},
+		bind.Object{Id: s.ccipObjects.CCIPObjectRefObjectId},
+		bind.Object{Id: s.ccipOfframpObjects.StateObjectId},
+		bind.Object{Id: s.ccipOfframpObjects.OwnerCapId},
+		s.mcmsPackageID,
+	)
+	require.NoError(s.T(), err, "transferring ownership of CCIP OffRamp to MCMS")
+	require.NotEmpty(s.T(), tx, "Transaction should not be empty")
+
+	// 2. Proposal execution with acceptance from MCMS (through bypasser)
+	input := ownershipops.AcceptCCIPOwnershipInput{
+		// MCMS related
+		MCMSPackageId:     s.mcmsPackageID,
+		MCMSStateObjId:    s.mcmsObj,
+		MCMSTimelockObjId: s.timelockObj,
+		MCMSAccountObjId:  s.accountObj,
+		MCMSRegistryObjId: s.registryObj,
+
+		CCIPPackageId: s.ccipPackageId,
+		CCIPObjectRef: s.ccipObjects.CCIPObjectRefObjectId,
+
+		RouterPackageId:     s.ccipRouterPackageId,
+		RouterStateObjectId: s.ccipRouterObjects.RouterStateObjectId,
+
+		OnRampPackageId:     s.ccipOnrampPackageId,
+		OnRampStateObjectId: s.ccipOnrampObjects.StateObjectId,
+
+		OffRampPackageId:     s.ccipOfframpPackageId,
+		OffRampStateObjectId: s.ccipOfframpObjects.StateObjectId,
+
+		// Proposal
+		Role: suisdk.TimelockRoleBypasser,
+
+		ChainSelector: uint64(s.chainSelector),
+	}
+	acceptOwnershipProposalReport, err := cld_ops.ExecuteSequence(s.bundle, ownershipops.AcceptCCIPOwnershipSeq, s.deps, input)
+	s.Require().NoError(err, "executing ownership acceptance proposal sequence")
+
+	timelockProposal := acceptOwnershipProposalReport.Output
+
+	// 3. Execute transfer ownership from original owner
+	// 3.1. Execute the proposal
+	s.ExecuteProposalE2e(&timelockProposal, s.bypasserConfig, 0)
+
+	// 3.2. Finish the ownership transfer with the original owner signer
+	executeTransferInput := ownershipops.ExecuteOwnershipTransferToMcmsSeqInput{
```

Comment:
@rodrigombsoares I changed this to use the execute sequence to transfer the ownership